pygmt/clib/session.py

```diff
@@ -1375,6 +1375,8 @@ def virtualfile_from_data(
         extra_arrays : list of 1d arrays
             Optional. A list of numpy arrays in addition to x, y and z. All
             of these arrays must be of the same size as the x/y/z arrays.
+        required_z : bool
+            State whether the 'z' column is required. [Default is False].
```

Comment:
I don't think we need to document the default values for these Python parameters, because the default values are already indicated in the function definitions.
```suggestion
            State whether the 'z' column is required.
```